index.json

```diff
@@ -3352,8 +3365,10 @@
   "vuiy.pw",
   "vuzimir.cf",
   "vvx046q.com",
+  "vw-audi.ml",
   "vw-seat.ml",
   "vw-skoda.ml",
+  "vyble.net",
```

Comment:
vyble.net seems to have google mail MX records - you sure it's (still) associated with a disposable domain service?